teamtwo/accounts/templates/registration/password_reset_done.html

```diff
@@ -0,0 +1,6 @@
+{% block title %}Email Sent{% endblock %}
```

Comment:
The file name is too similar to other password_reset related files.
I suggest changing the file name from "password_reset_done.html" to "reset_email_sent.html"